input/tex/extensions/ams.rst

```diff
@@ -38,4 +38,21 @@ remove it:
      tex: {packages: {'[-]': ['ams']}}
    };
 
+
+-----
```

Comment:
There are now options for the ams extension.  Can those be added here?

Reply:
Added the descriptions.